src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -595,7 +597,7 @@ AthensSurfaceExamples class >> exampleDrawText [
 		can pathTransform translateX: 0 Y: (font getPreciseAscent).
 		can drawString: 'Hello Athens!' ].
 	
-	Display getCanvas drawImage: surf asForm at: 0@0
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?